lib/rspec/core/example.rb

```diff
@@ -138,7 +155,9 @@ def initialize(example_group_class, description, user_metadata, example_block=ni
         @example_block       = example_block
 
         @metadata = Metadata::ExampleHash.create(
-          @example_group_class.metadata, user_metadata, description, example_block
+          @example_group_class.metadata, user_metadata,
```

Comment:
I always think multiple comma separated args on the same line is a smell, if one of them changes it'll churn the line with both of them making it less obvious in the diff what happened, how about
  
```
@metadata = Metadata::ExampleHash.create(
@example_group_class.metadata, 
  user_metadata,
  example_group_class.method(:next_runnable_index_for),
  description,
  example_block,
)
```

Note the trailing comma, which means that if we add another argument, that line won't churn.